scripts/release/debian/build.sh

```diff
@@ -21,7 +21,8 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # Update APT packages
 apt-get update
 # uuid-dev is used to build _uuid module: https://github.com/python/cpython/pull/3796
-apt-get install -y libssl-dev libffi-dev python3-dev zlib1g-dev uuid-dev wget debhelper
+# libbz2-dev is used to install bz2 module: https://github.com/Azure/azure-cli/pull/32163
+apt-get install -y libssl-dev libffi-dev python3-dev zlib1g-dev uuid-dev wget debhelper libbz2-dev
```

Comment:
How about RPM? Does the system Python installed by RPM already have `bz2` support?